content/en/real_user_monitoring/guide/setup-rum-deployment-tracking.md

```diff
@@ -0,0 +1,188 @@
+- P75 for Core Web Vitals
+
+These widgets can be exported to dashboards and monitors.
+
+{{< img src="real_user_monitoring/guide/setup-rum-deployment-tracking/browser-rum-app-overview-deployment-tracking.png" alt="Browser Deployment Tracking in RUM Application Overview" style="width:100%;">}}
+
+
+### Deployment Comparison
+
+Click on any version row in the List of Versions table to open a version comparison page, allowing you to compare two versions of the same service. By default, the selected version will be compared to all previous versions but you can change it to compare any two versions within the past 30 days.
+
+Similar to the graphs on the Application Overview page, User Sessions, Core Web Vitals, and Errors graphs show an overview of a deployment rollout or spikes in error rates. On this page, the graphs highlight the selected versions for comparison and leave all other versions in gray for additional context.
+
+As you are monitoring your release, this makes it easy for you to compare the performance of code deployments against existing live code to verify that new code is performing properly and that no new errors have surfaced in between versions. 
+
+{{< img src="real_user_monitoring/guide/setup-rum-deployment-tracking/browser-deployment-tracking-comparison.png" alt="Browser Deployment Tracking Comparison" style="width:75%;">}}
+
+The Issues tab, lists differences in errors detected for each the two versions, highlighting:
+- Error Count by Version 
+- % of Views with Errors by Version 
+- Error Tracking issues
+
+{{< img src="real_user_monitoring/guide/setup-rum-deployment-tracking/browser-deployment-tracking-comparison-error.png" alt="Browser Deployment Tracking Comparison Errors" style="width:75%;">}}
+
+### Explore the RUM Deployment Tracking Powerpacks
+You can quickly add deployment tracking for your RUM services to dashboards using the powerpacks menu on a dashboard and searching for the “Deployment Version Tracking” powerpack. You can then iterate and add any other widget to your dashboards to help your teams release new features safely.
+
+{{< img src="real_user_monitoring/guide/setup-rum-deployment-tracking/browser-deployment-tracking-powerpack.png" alt="Browser Deployment Tracking Powerpack" style="width:75%;">}}
+
+{{% /tab %}}
+{{% tab "Mobile RUM" %}}
+
+### Using version tags on the Application Overview page
+
+An application configured with version tags has a Deployment Tracking section on the Application Overview page. The Deployment Tracking section shows all versions of the application and services that were active during the selected time interval. 
+
+This enables you to adopt modern deployment strategies with peace of mind by letting you quickly roll back release candidates as soon as you spot an issue so you can avoid negative user experiences. These out of the box graphs are aggregated across versions, making it easy to spot problems in your application before they turn into serious issues.
```

Comment:
We should add the tabs after this section, since they're identical and the content is being duplicated. Duplication like this adds a lot of overhead for us over the life of the doc.

Reply:
the reason that i added the tabs before this is that the entire section has some variability for web vs. mobile apps. there are some sections that are duplicated but we would be adding a much of tabs throughout the sections. Wasn't sure if that made sense from a readability standpoint